### DISCUSSION TITLE
How to set Ctrl+/ as a keybind?

### DISCUSSION THREAD

**Discussion by cyrus01337** [neovim/neovim#29621] [Q&A] [answered]
New to and loving Neovim so far, I come from VS Code and wanted to rebind `gcc` (toggle line comment) to `Ctrl+/`, I tried using `<C-/>` and `<C-//>` to no avail, and couldn't find the help page for viewing keybinds.

So far the following works thanks to #29075:
```lua
vim.keymap.set("n", "<leader>c", "gcc", { remap = true })
```

**Comment by zeertzjq** [comment, marked as answer]
Try `<C-_>` instead.

**Reply by cyrus01337** [reply]
Works perfectly, thank you so much! :revolving_hearts:
